VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -167,6 +177,20 @@ public void download(DownloadEntry videoData) {
         }
     }
 
+    private boolean hasConnectivity(CourseBaseActivity activity) {
```

Comment:
This should probably be a global utility method (maybe in `NetworkUtil`), as this can be used from other components as well.